### PR TITLE
dpinger STDEV GUI Additions

### PR DESCRIPTION
**Pull request by marjohn56**
Hide STDEV from GUI when Dpinger not enabled

### CODE REVIEW

File: src/www/system_gateways_edit.php [hunk 1]
```diff
@@ -37,9 +37,9 @@
     $a_gateways_arr[] = $gw;
```
Comment (fichtner) [inline src/www/system_gateways_edit.php]: you can drop this whole file from the PR

File: src/www/widgets/widgets/gateways.widget.php [hunk 2]
```diff
@@ -78,6 +102,11 @@ function gateways_widget_update(sender, data)
         <tr>
             <th><?=gettext('Name')?></th>
             <th><?=gettext('RTT')?></th>
+<?php
+            if (isset($config['system']['prefer_dpinger'])) :?>                         
+            <th><?=gettext('STDEV')?></th>
```
Comment (fichtner) [inline src/www/widgets/widgets/gateways.widget.php]: Should it be STDEV or RTTd / RTTD consistently for the visible GUI labels ?

Comment (marjohn56) [reply src/www/widgets/widgets/gateways.widget.php]: Well my original.. original... was RTTd, then someone said it should be STDEV... so which would you like, Personally I prefer RTTd

Comment (fichtner) [reply src/www/widgets/widgets/gateways.widget.php]: Word! Let's make it RTTd then.

File: src/www/system_gateways_edit.php (hunk 1 above)
Comment (marjohn56) [reply src/www/system_gateways_edit.php]: What ,,, so it becomes

$a_gateways_arr = array();
$a_gateways = return_gateways_array(true, false, true);
if (!isset($config['system']['prefer_dpinger'])) {
...

Comment (fichtner) [reply src/www/system_gateways_edit.php]: it's ok as is now: return_apinger_defaults is a cheap call and is fixed up when prefer_dpinger is set below

the diff later also rewrites part of unrelated fixes on master

File: src/www/widgets/widgets/gateways.widget.php (hunk 2 above)
Comment (marjohn56) [reply src/www/widgets/widgets/gateways.widget.php]: So are you fixing this PR or do you want me to make any changes?

Little confused by the comment above

"it's ok as is now: return_apinger_defaults is a cheap call and is fixed up when prefer_dpinger is set below
the diff later also rewrites part of unrelated fixes on master"

Better I leave you to finish it if it's affecting other files.

**Comment by fichtner** [comment]
@marjohn56 remove all modifications from src/www/system_gateways_edit.php from this PR -- the state in this PR reverts master state in this file and it already works with prefer_dpinger :)

**Comment by marjohn56** [comment]
Slight screw up with the merge for some reason. Reverted it - there should now only be 3 files in the commit 6c61f85.

If you can tell me how to get rid of those merge commits I will.

**Comment by fichtner** [comment]
No need, squashed for maximum effect. 👍

**Pull request merged**